### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by kryvets-vadym**
[DEMO LINK](https://kryvets-vadym.github.io/react_list-of-goods-hooks/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -15,47 +16,111 @@ export const goodsFromServer = [
...
+  if (isReversed) {
+    visibleGoods.reverse();
+  }
+
+  return visibleGoods;
+}
+
 export const App: React.FC = () => {
+  const [isReversed, setReverse] = useState<boolean>(false);
+  const [sortType, setSortType] = useState<SortType>(SortType.NONE);
```
Comment (spiritkyiv) [inline src/App.tsx:58]: interest

**Review by spiritkyiv — APPROVED** [review]
looks good!

File: src/App.tsx (hunk 1 above)
Comment (Olexandr6) [inline src/App.tsx:58]: You don't need to define type explicitly here
```suggestion
  const [isReversed, setReverse] = useState(false);
  const [sortType, setSortType] = useState(SortType.NONE);
```

**Review by Olexandr6 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**